### ISSUE TITLE
SELECT INTO locks MSSQL database

### ISSUE THREAD

**Issue by ohadle** [sqlalchemy/sqlalchemy#4599]
Also posted [on SO](https://stackoverflow.com/questions/55514844/select-into-from-sqlalchemy-locks-database).

When executing a `SELECT INTO` query for SQL Server through `SQLAlchemy` execution finishes, but the database locks (and the query doesn't do what it should).

I tried with `engine.execute()`, `connection.execute()` and using transactions.

```python
query = f'''SELECT a, b
            INTO DB.schema.sometable
            FROM othertable;'''
print(query)

engine = create_engine("mssql+pyodbc://:@DB")
con = engine.connect()
trans = con.begin()
try:
    res = con.execute(query)
    trans.commit()
except:
    trans.rollback()
    raise
```

Running the same query with `SSMS` succeeds.

**Comment by zzzeek** [comment]
hi there -

can you please describe what "The database locks" means as well as what "the query does not do what it should"?   What actually happens please ?

**Comment by ohadle** [comment]
OK apparently I made a mistake - the transaction code I posted actually works!

For posterity:
Running the `SELECT INTO` query with `engine.execute()` just didn't seem to work - when I try to `SELECT` from the destination table I insert into with SSMS I get that the table doesn't exist. 
If I run the same query with `connection.execute()`, and try to `SELECT` from the destination table I just get stuck on `Executing query...`.

Thanks!
